custom-metrics-stackdriver-adapter/examples/direct-to-sd/custom-metrics-sd.yaml

```diff
@@ -19,18 +19,23 @@ spec:
       - command:
         - /bin/sh
         - -c
-        - ./sd_dummy_exporter --metric-name=foo --metric-value=40 --pod-id=$(POD_ID)
-        image: gcr.io/google-containers/sd-dummy-exporter:v0.1.0
+        - ./sd_dummy_exporter --metric-name=foo --metric-value=40 --use-new-resource-model --use-old-resource-model --pod-name=$(POD_NAME) --namespace=$(NAMESPACE)
+        image: gcr.io/google-containers/sd-dummy-exporter:v0.2.0
         name: sd-dummy-exporter
         resources:
           requests:
             cpu: 100m
         env:
-          - name: POD_ID
```

Comment:
The same, here you should also specify POD_ID because it is required for old model.

Reply:
Done